src/app/app.component.ts

```diff
@@ -0,0 +1,29 @@
+import { TranslateService } from '@ngx-translate/core';
+import { ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { LoaderService } from './core/services/loader.service';
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
```

Comment:
According to best practices, it better to use onPush strategy ( changeDetection: ChangeDetectionStrategy.OnPush) for each component. That helps you to prevent performance issue with application.